.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: ''
+labels: bug
+assignees: ''
+---
+
+## Description
+<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Actual Behavior
+<!--- Tell us what happens instead -->
+
+## Steps to Reproduce
+<!--- Provide an repeatable set of steps to reproduce this bug -->
+1.
+2.
+3.
+4.
+
+## Context
+<!--- How has this bug affected you? What were you trying to accomplish? -->
+
+## Possible Fix
+<!--- Not obligatory, but suggest a fix or reason for the bug -->
+
+## Logs
+<!--- Error logs in any form are really helpful -->
+
+## Your Environment
```

Comment:
I think you can suggest to place here for example output of 'casadm -P', 'casadm -L' and 'lsblk' commands